test/calibration/experiments/test_drag.py

```diff
@@ -96,6 +97,27 @@ def test_end_to_end(self):
         self.assertTrue(abs(result.value.n - backend.ideal_beta) < self.test_tol)
         self.assertEqual(result.quality, "good")
 
+    @data(
+        (0.0050, 1.0, 0.00, [1, 3, 5], None),  # partial oscillation.
+        (0.0025, 0.5, 0.00, [1, 3, 5], None),  # even slower oscillation with amp < 1
+        (0.0050, 0.8, 0.05, [3, 5, 7], None),  # constant offset, i.e. lower SNR.
+        (0.1000, 0.5, 0.10, [1, 3, 5], np.linspace(-1, 1, 51)),  # Beta not in range
+    )
```

Comment:
Can we add one more case like `(0.3500, 0.5, 0.10, [1, 3, 5], np.linspace(-2.5, 2.5, 51))` that puts a maximum closer to zero than the ideal beta is?